go/host/rpc/clientapi/client_api_obscuroscan.go

```diff
@@ -97,7 +97,7 @@ func (api *ObscuroScanAPI) GetLatestTransactions(num int) ([]gethcommon.Hash, er
 		for _, txHash := range batchTxHashes {
 			txHashes = append(txHashes, txHash)
 			if len(txHashes) >= num {
-				break
```

Comment:
This `break` was only breaking the inner loop, and not the outer loop, potentially leading to huge numbers of txs being returned.

Reply:
oops